pkg/oomstore/types/apply/group.json

```diff
@@ -0,0 +1,26 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/oomstore/types/feature.json",
+    "$title": "apply feature json schema",
+    "description": "apply feature json schema",
+    "type": "object",
+    "properties": {
+        "kind": {
+            "description": "must be group",
+            "type": "string"
+        },
+        "name": {
+            "description": "name of the group",
+            "type": "string"
+        },
+        "category": {
+            "description": "group category, which can only be batch or stream",
+            "type": "string"
```

Comment:
better be enum?

Reply:
Yes, I just wrote it here for now, I'll change it later

Reply:
done